### PR TITLE
Release 5.6.26-74.0

### PR DESCRIPTION
**Pull request by tplavcic**
This is a release branch for version 5.6.26-74.0 with appropriate tag.

### CODE REVIEW

**Comment by laurynas-biveinis** [comment]
Do PerconaFT (and backup plugin) have their own release branches? If so, they must be merged to their trunks first

**Comment by tplavcic** [comment]
Yes they do, I'll create PR's for those but will wait for George with the merge just to be sure.
Will ping when ready to complete PS branch merge.

**Comment by tplavcic** [comment]
@laurynas-biveinis The release branches in PerconaFT and Percona-TokuBackup don't have any commits that are already not on master branches.
So I cannot create a PR there and merge them, and when I merge from command line I get "Already up-to-date."
Since the commits for release are tagged I think those branches are just obsolete and can be deleted, but @georgelorchpercona wants to keep them and I can't blame him :)
Basically I think we're good to go for PS release branch merge to 5.6

**Comment by laurynas-biveinis** [comment]
LGTM, I will try to handle the merge of this to trunk so that the right thing happens
